caffeine/src/test/java/com/github/benmanes/caffeine/cache/AsMapTest.java

```diff
@@ -2309,6 +2310,18 @@ public void keySpliterator_estimateSize(Map<Int, Int> map, CacheContext context)
     assertThat(spliterator.estimateSize()).isEqualTo(context.initialSize());
   }
 
+  @CacheSpec
+  @CheckNoStats
+  @Test(dataProvider = "caches")
+  public void keySpliterator_characteristics(Map<Int, Int> map, CacheContext context) {
```

Comment:
We should probably add the same test cases to `AsyncAsMapTest` too. Unfortunately a lot of similarity between the tests but given different parameters as async/sync views (`Map<K, CompletableFuture<V>>` vs `Map<K, V>`).

Reply:
yeah it does but their different views (`Cache.asMap()` & `AsyncCache.synchronous().asMap()` vs `AsyncCache.asMap()`). Thanks for reporting and jump on the fix!

Reply:
Ah that makes sense.